_includes/de_code_of_conduct.html

```diff
@@ -0,0 +1,29 @@
+<h2>Code of Conduct</h2>
+
+All events are covered under the <a href="https://berlincodeofconduct.org/de" target="_blank">Berlin Code of Conduct</a>.
```

Comment:
German translation is still needed for Lines 1-4 @pascalwengerter @kajatiger @ferbsx

Reply:
"Code of Conduct" could stay English, cause it's also known under that name in German. Otherwise the correct translation would be "Verhaltenskodex". 
For line 3 it would be:
"Auf allen Veranstaltungen gilt der Berliner Verhaltenskodex".
You can spell the German word _Kodex_ with a **K** or with a **C** in the beginning. As you like.